py/tests/algorithms/anomaly_likelihood_test.py

```diff
@@ -154,7 +153,7 @@ def testCalcSkipRecords(self):
 
 
   def testHistoricWindowSize(self):
-    l = an.AnomalyLikelihood(claLearningPeriod=2,
+    l = an.AnomalyLikelihood(learningPeriod=2,
```

Comment:
what is this change based on? Note that Likelihood in python is just copied from nupic

Reply:
I just based it on this piece of code: https://github.com/htm-community/nupic.cpp/blob/69555d7396d05c4af1e02199141d047963e3ccfc/py/htm/algorithms/anomaly_likelihood.py#L183-L187

I think that this unit test is just using old naming